### ISSUE TITLE
Upgrading VS build gives CS1668 until reboot

### ISSUE THREAD

**Issue by danmoseley** [dotnet/roslyn#57143]
1. Update VS build (dogfooding)
2. Without rebooting or reopening the command prompt (because i want to preserve state in it), build the tree, and get this warning for every project:

```
CSC : warning CS1668: Invalid search path 'C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Tools\MSVC\14.30.30528\lib\x86' specified in 'LIB environment variable' -- 'directory does not exist' [C:\git\runtime\src\libraries\System.Diagnostics.EventLog\src\Messages\System.Diagnostics.EventLog.Messages.csproj]
```

It's no surprise because VS updated my tools version. My guess is that in 2021 the LIB is of little value to the C# compiler? Would it be reasonable to remove this warning?
